content/docs/engine-dev/tools/tiaf/maintaining-tiaf/debugging-tiaf.md

```diff
@@ -77,47 +87,179 @@ In the log above, you can extract the following information about the AR run and
-- The live Python test runner was used (this feature is deprecated).
-- The arguments used to invoke the Python runtime were as follows:
+- The arguments used to invoke the native runtime were as follows:
 
     ```
-    --sequence=tia --fpolicy=continue --suites=main,smoke --labelexcludes=REQUIRES_gpu --changelist=D:\workspace\o3de\build\windows\bin\TestImpactFramework\profile\Temp\Python\changelist.76aeff41529f498297bf8194d38d5414.json --report=D:\workspace\o3de\build\windows\bin\TestImpactFramework\profile\Temp\Python\Reports\report.76aeff41529f498297bf8194d38d5414.json --targetout=stdout --testrunner=live
+    --sequence=tianowrite --fpolicy=continue --suites=main,smoke --labelexcludes=REQUIRES_gpu --ipolicy=continue --changelist=D:\workspace\o3de\build\windows\bin\TestImpactFramework\profile\Temp\Native\changelist.06011392e5a7456e817cda03f8f8c606.json --report=D:\workspace\o3de\build\windows\bin\TestImpactFramework\profile\Temp\Native\Reports\report.06011392e5a7456e817cda03f8f8c606.json --targetout=stdout
     ```
 
 To reproduce this particular AR run, perform the following steps:
 
-1. Check out the commit `59e24a62859ce8a932e338f36432e3228621c1ec`.
-2. Use the [storage query tool](./storage-query-tool.md) to retrieve the relevant historic data for this run.
-3. Build the commit.
-4. Place the historic data in the local historic data folder (see the historic workspace entry in any of the `tiaf.json` files in the TestImpactFramework folder in your build directory).
-5. Run the TIAF AR script with the the following arguments (omitting the S3 bucket so that local persistent storage is used):
+1. Recreate the historic data locally.
+2. Build and run the commit failing in AR.
+3. Run the TIAF runtime with a debugger attached.
+
+### Step 1: Recreate the historic data locally
+
+To recreate the historic data locally, you can either use the [storage query tool](./storage-query-tool.md) to retrieve the relevant historic data for this run from the server running AR and place the contents in the `<build_path>/bin/TestImpactFramework/debug/Persistent/Native/historic/main-smoke` directory, or recreate the historic data from scratch. As the latter approach is the most reliable (as it does not require access to the persistent storage used by AR), this is the approach that will be described below.
+
+Checkout and build the commit used by the last run (in this instance `4b65521188a2a0beb399eca7b692216d2fe3d209`). For native runs, it is recommended you build the `debug` configuration to make debugging easier but for Python runs you must build the `profile` configuration as, at the time of writing, Python tests will not execute properly with the `debug` build configuration.
+
+Run the TIAF using the AR scripts to generate the historic data used in the failing AR run locally. To do so, run the `tiaf_driver.py` script and supply the arguments used in the AR run. For this demonstration, Visual Studio Code will be used. 
```

Comment:
Could you replace the `tiaf_driver.py` inline code snippet with a hyperlink to the file to locate help users locate it within the engine?.

```suggestion
Run the TIAF using the AR scripts to generate the historic data used in the failing AR run locally. To do so, run the [tiaf_driver.py](https://github.com/o3de/o3de/blob/development/scripts/build/TestImpactAnalysis/tiaf_driver.py) script and supply the arguments used in the AR run. For this demonstration, Visual Studio Code will be used. 
```